fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryLoadByKeysPlan.java

```diff
@@ -159,8 +160,8 @@ public String toString() {
 
     @Nonnull
     @Override
-    public Set<CorrelationIdentifier> getCorrelatedTo() {
-        return ImmutableSet.of(); // TODO this should be reconsidered when we have selects
```

Comment:
This `TODO` got dropped. I'm not sure I totally follow what it was trying to say, though. Do we still need it, or was it already out of date/irrelevant?